### ISSUE TITLE
Tasmota firmware answered a wrong formatted JSON string

### ISSUE THREAD

**Issue by rickyisanerd** [reloxx13/TasmoAdmin#369]
Problem/Motivation
When click the light it turns on just fine, but when I try to via wifi i get this error message.

(Why the issue was filed)
One of my recent switches has this error, said to copy paste it as issue on github; Somehow the Tasmota firmware answered a wrong formatted JSON string which needs to get adjusted in TasmoAdmin. JSON Answer => File/cm?user=admin&password=admin&cmnd=status+0 not_found

Expected behavior
(What you expected to happen)
I expected the light to switch on and off

Actual behavior
Gives me an error

(What actually happened)

Steps to reproduce
Try to turn on or off light via wifi

(How can someone else make/see it happen)
Have my sh*tty luck

Proposed changes
I am assuming it is a glitch of some sort and I can just change a GPIO or issue a command. I think power outage caused this but not sure.

![Screenshot 2020-04-06 15 07 28](https://user-images.githubusercontent.com/8869359/78602215-af605e80-781b-11ea-9e04-59d95a99f0c6.png)

**Comment by Exezz78** [comment]
Hi rickyisanerd,

had a similar issue yesterday caused by a FritzRepeater (IP-Addresses were used in double - one for the switch and the same for the repeater).

Have you checked if the IP might have changed and therefor the switch is not available in TasmoAdmin? Have a look in the router interface and try opening the Switch-IP directly.

I removes the "wrong" entry in TasmoAdmin and added the new via Autoscan. 

Hope this helps.

Regards,
Exezz

**Comment by rickyisanerd** [comment]
Hi Exezz,I figured out that it was due to the switch being 3 pole, I didn't know it was controlled by another switch in the house.  So when you flip the other switch this Tasmota one doesn't know what to do and from what I can see just resets.I was able to log into the device using the tasmota wifi ssid and reassign to my wifi but I can't use it.I think I am going to swap this wifi switch with the other and see if that helps.Thank You,Ricky CarterSent from my T-Mobile 4G LTE Eraser
-------- Original message --------From: Exezz78 <notifications@github.com> Date: 4/29/20  1:49 AM  (GMT-06:00) To: reloxx13/TasmoAdmin <TasmoAdmin@noreply.github.com> Cc: Ricky <rickyisanerd@gmail.com>, Author <author@noreply.github.com> Subject: Re: [reloxx13/TasmoAdmin] Tasmota firmware answered a wrong formatted
  JSON string (#369) 
Hi rickyisanerd,
had a similar issue yesterday caused by a FritzRepeater (IP-Addresses were used in double - one for the switch and the same for the repeater).
Have you checked if the IP might have changed and therefor the switch is not available in TasmoAdmin? Have a look in the router interface and try opening the Switch-IP directly.
I removes the "wrong" entry in TasmoAdmin and added the new via Autoscan.
Hope this helps.
Regards,
Exezz

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/reloxx13/TasmoAdmin/issues/369#issuecomment-621021378",
"url": "https://github.com/reloxx13/TasmoAdmin/issues/369#issuecomment-621021378",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]
